secp256k1.py

```diff
@@ -238,6 +238,7 @@ def mul(self, ps):
 SECP256K1_G = (0x79BE667EF9DCBBAC55A06295CE870B07029BFCDB2DCE28D959F2815B16F81798, 0x483ADA7726A3C4655DA4FBFC0E1108A8FD17B448A68554199C47D08FFB10D4B8, 1)
 SECP256K1_ORDER = 0xFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFEBAAEDCE6AF48A03BBFD25E8CD0364141
 SECP256K1_ORDER_HALF = SECP256K1_ORDER // 2
+NUMS_POINT = SECP256K1.lift_x(0x50929b74c1a04954b78b4b6035e97a5e078a5a0f28ec96d547bfee9ace803ac0)
```

Comment:
as far as i understand, `lift_x` converts the bytes into an actual EC point (with an x and y coordinate). we could just encode nums as the x-only bytes, i.e. call it `NUMS_H = 0x590...`. this works because we don't actually need NUMS for any EC calculations (like adding points, etc). we just want to see if NUMS was used as an internal key, which the x-only encoding should be sufficient for.

this would make the check easier, in that we could just do `internal_key == NUMS_H`, since we are just comparing bytes at that point.

Reply:
Resolved. See https://github.com/S3RK/bip-352/pull/3/commits/24a64917fcc66cdaa4ba18273468e9a0333d399f